reaper.go

```diff
@@ -62,6 +67,32 @@ func (r *agentReaper) unlock() {
 	r.RUnlock()
 }
 
+func (r *agentReaper) getEpoller(pid int) (*epoller, error) {
```

Comment:
Nit: you could drop the `get` prefix - it's descriptive but not necessary according to:

- https://golang.org/doc/effective_go.html#Getters

Reply:
Hi @jodh-intel , yes, this "get" prefix is redundant, but in order to keep align with the reaper's
other methods such as getExitCodeCh, Can I  keep it here?

Reply:
Hi @lifupan - Fine with me - any "nits" are just tiny issues that shouldn't block the PR landing imho ;)